src/test/java/org/ays/common/util/AysDataProvider.java

```diff
@@ -175,18 +175,19 @@ public static Object[][] invalidFirstAndLastNamesDataForAssignment() {
     @org.testng.annotations.DataProvider(name = "invalidRejectReason")
     public static Object[][] invalidRejectReason() {
```

Comment:
@cagla88 selam emegine saglik gayet sade ve anlasilir olmus tum duzeltmeler bence. Sadece bu kismi anlamadim ilk baktigimda, o yuzden bu isimlendirmeyi  invalidRejectReasonInputsForAdminRegistration ya da  invalidRejectReasonInputs seklinde degistirsek nasil olur ?

Reply:
@htczrrsz selam, teşekkür ederim. Bu kısımda bi değişiklik yapmamıştım ama evet haklısın daha anlaşılır olması için düzenlerim

Reply:
@cagla88 evet evet bir degisiklik yapilmadigini goruyorum o kısımda zaten ama incelerken o kısmı da görünce halihazırda da uzerinde çalışılıyorken düzeltilebilir diye belirtmek istedim ☺️  teşekkürler eline sağlık tekrar.